### DISCUSSION TITLE
I was able to add 👀 twice

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#51839] [Mobile]
GitHub Mobile beta

I opened this from an email, and then added the 👀. Something happened and the emoji wasn't present, so I added them again:

![Screenshot_20230403-090157.png](https://user-images.githubusercontent.com/2119212/229517808-b59802eb-5783-45ab-abd7-6105cdc5b8d6.png)

**Comment by iskae** [comment]
Hello @jsoref 👋 

Thanks a bunch for reaching out ❤️ 

I was unfortunately not able to reproduce this issue following the steps that you have described 😅 Were you able to constantly reproduce this issue?

**Reply by jsoref** [reply]
No. It was a one-off and I wasn't screen recording/taking meticulous notes of what I was doing. It's quite likely there were a couple of other steps necessary. Sorry.

**Reply by iskae** [reply]
It is most probably related to our optimistic updates, which we are slowly moving towards for a more stable solution 👍 It is highly unlikely this will happen in the near future, but if it happens and by any chance you were able to reproduce it always we would love to solve the issue 🙇
